### DISCUSSION TITLE
[IntelliSense]  Why do you have to press the space bar to display code prompts?

### DISCUSSION THREAD

**Discussion by MaybeQHL** [tailwindlabs/tailwindcss#9445] [Ideas]
I want to type out the first letter so I can see the hint.

**Comment by ivanjeremic** [comment]


**Reply by MaybeQHL** [reply]
> I know this annoys me for a long time now and still no one has a fix. :(

Does the author explain why?

**Comment by MaybeQHL** [comment]
I have solved the problem. 
![image](https://user-images.githubusercontent.com/34638673/193176459-22bc6bcb-8fab-48bf-b5a1-33f1497325dc.png)
